components/rmrk/Gallery/Search/SearchBar.vue

```diff
@@ -122,6 +122,10 @@
         ticks
         @change="sliderChange">
       </b-slider>
+      <span v-if="sliderDirty"
+        >Prices ranging from {{ this.query.priceMin / 1000000000000 }} to
```

Comment:
? oh no

Reply:
I suppose there's way more elegant solution to this? Which part of is it wrong? All of it?

Reply:
1e12 (10 ** 12) works only for KSM.
It works for now but I think it would be a problem with other implementations (BSX)

Reply:
The correct way is to use chain decimals saved in store. For refference check `support.ts`